bonito/openvino/model.py

```diff
@@ -0,0 +1,93 @@
+import os
+import numpy as np
+import torch
+
+try:
+    from openvino.inference_engine import IECore, StatusCode
+    from .loader import torch2openvino
+except ImportError:
+    pass
+
+class OpenVINOModel:
+
+    def __init__(self, model, half, dirname):
+        self.model = model
+        self.alphabet = model.alphabet
```

Comment:
what is it?

Reply:
The are the classification labels predicted at each timestep by the model.

Reply:
Good. I simply don't see any usage of them in this particular class. Means it passes through this data. Looks excessive

Reply:
This is an attribute required for `evaluate.py` but located in `model` object